src/BackpackServiceProvider.php

```diff
@@ -43,6 +44,7 @@ public function boot(\Illuminate\Routing\Router $router)
         $this->loadTranslationsFrom(realpath(__DIR__.'/resources/lang'), 'backpack');
         $this->loadConfigs();
         $this->registerMiddlewareGroup($this->app->router);
+        $this->registerMacros();
```

Comment:
I'd move the macro to a separate file, which we can easily include here (first line in this method). Just like we have `helpers.php` we could have `macros.php`:

```suggestion
        include(__DIR__.'/macros.php');
```

Reply:
Alternatively, maybe we can load the `macros.php` file the same way we do `helpers.php`, in the `composer.json` file.

Reply:
Agree, will make the change! 👍